draft-ietf-bidding-and-auction-services.md

```diff
@@ -935,6 +938,8 @@ response from Bidding and Auction Services. It takes as input the
    return failure.
 1. Let `processed response` be a new structure analogous to
    [server auction response](https://wicg.github.io/turtledove/#server-auction-response).
+1. If `response["nonce"]` exists and is a valid [UUID], set
+   `processed response["nonce"]` to `response["nonce"]`.

```

Comment:
do we change `response["nonce"]` to lowercase before setting to processed response? https://source.chromium.org/chromium/chromium/src/+/main:content/browser/interest_group/bidding_and_auction_response.cc;l=115?q=output.nonce%20%3D%20nonce_uuid.A&sq=&ss=chromium